### PR TITLE
Update to airbase 156 + other deps

### PR DESCRIPTION
**Pull request by wendigo**

### CODE REVIEW

**Comment by wendigo** [comment]
cc @findepi

File: pom.xml [hunk 1]
```diff
@@ -92,6 +92,8 @@
         <dep.airlift.version>247-SNAPSHOT</dep.airlift.version>
         <dep.bouncycastle.version>1.78.1</dep.bouncycastle.version>
         <dep.bytebuddy.version>1.14.14</dep.bytebuddy.version>
+        <!-- TODO: remove when promoted to Airbase -->
+        <dep.checkstyle.version>10.16.0</dep.checkstyle.version>
```
[the remark this answers is not in the document]
Comment (wendigo) [reply pom.xml:96]: I'll but airbase takes a long time in reviews

**Comment by findepi** [comment]
LGTM % https://github.com/airlift/airlift/pull/1149#discussion_r1582839568. 
let's update checkstyle in airbase (unless there is some reason for urgency to do otherwise)

**Comment by wendigo** [comment]
@findepi I can update this in airbase. Btw. when I've updated this in Trino - there are some previously unreported issues

File: pom.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (wendigo) [reply pom.xml]: https://github.com/airlift/airbase/pull/408

[the remark this answers is not in the document]
Comment (wendigo) [reply pom.xml]: Dropped checkstyle & added one more cleanup commit

File: pom.xml [hunk 2]
```diff
@@ -297,13 +297,6 @@
                 <version>${dep.jjwt.version}</version>
             </dependency>
 
-            <!-- TODO: move to airbase -->
-            <dependency>
-                <groupId>jakarta.servlet</groupId>
-                <artifactId>jakarta.servlet-api</artifactId>
-                <version>6.0.0</version>
```
Comment (findepi) [inline pom.xml:304]: Shouldn't this be part of `Update airbase to 155` commit?

File: pom.xml [hunk 3]
```diff
@@ -91,7 +91,7 @@
 
         <dep.airlift.version>247-SNAPSHOT</dep.airlift.version>
         <dep.bouncycastle.version>1.78</dep.bouncycastle.version>
-        <dep.bytebuddy.version>1.14.13</dep.bytebuddy.version>
+        <dep.bytebuddy.version>1.14.14</dep.bytebuddy.version>
```
Comment (findepi) [inline pom.xml:94]: if Trino maintains its own byte buddy version, why would we move this version number in https://github.com/airlift/airbase/pull/408 to be controlled by airbase?

Comment (wendigo) [reply pom.xml:94]: It's shared between airlift and Trino - hence it should be defined in airbase

File: pom.xml (hunk 2 above)
Comment (wendigo) [reply pom.xml:304]: it was introduced earlier than 155 as far as I remember

File: pom.xml [hunk 4]
```diff
@@ -5,7 +5,7 @@
     <parent>
         <groupId>io.airlift</groupId>
         <artifactId>airbase</artifactId>
-        <version>153</version>
+        <version>155</version>
```
Comment (electrum) [inline pom.xml:8]: We can use 156 now

File: pom.xml (hunk 3 above)
Comment (electrum) [inline pom.xml:94]: Can remove this property with Airbase 156

**Review by electrum — APPROVED** [review]

File: pom.xml (hunk 2 above)
Comment (wendigo) [reply pom.xml:304]: Moved to `update to airbase 156` commit

File: pom.xml (hunk 4 above)
Comment (wendigo) [reply pom.xml]: Done.

File: pom.xml (hunk 3 above)
Comment (wendigo) [reply pom.xml]: Done.

**Comment by wendigo** [comment]
AC @findepi @electrum

**Comment by wendigo** [comment]
Let's wait for Jetty 12.0.9 (we need it)

**Review by electrum — APPROVED** [review]

**Pull request merged**